api4/user_test.go

```diff
@@ -1099,32 +1099,73 @@ func TestGetUsersByIds(t *testing.T) {
 	th := Setup().InitBasic()
 	defer th.TearDown()
 
-	users, resp := th.Client.GetUsersByIds([]string{th.BasicUser.Id})
-	CheckNoError(t, resp)
+	t.Run("should return the user", func(t *testing.T) {
+		users, resp := th.Client.GetUsersByIds([]string{th.BasicUser.Id})
 
-	if users[0].Id != th.BasicUser.Id {
-		t.Fatal("returned wrong user")
-	}
-	CheckUserSanitization(t, users[0])
+		CheckNoError(t, resp)
 
-	_, resp = th.Client.GetUsersByIds([]string{})
-	CheckBadRequestStatus(t, resp)
+		if users[0].Id != th.BasicUser.Id {
```

Comment:
nit: perhaps leverage `require` instead of `t.Fatal` for clearer error messages?